test/unit/spec/EnrollChoices_spec.js

```diff
@@ -186,24 +199,38 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
       return setup(res);
     }
 
-    function setupMultipleFactorEnrollments (factorName, status, enrollment, policyEnrollment, singleFactorRes) {
-      var res = singleFactorRes ? deepClone(singleFactorRes) : deepClone(resAllFactors);
-      var index = singleFactorRes ? 0 : factorEnrollList[factorName].index;
-      res.response._embedded.factors[index].status = status;
-      res.response._embedded.factors[index].enrollment = enrollment;
-      res.response._embedded.factors[index].policy.enrollment = policyEnrollment;
-      return setup(res);
+    function setupMultipleFactorEnrollments (options) {
+      var res = options.singleFactorRes ? deepClone(options.singleFactorRes) : 
+        options.useProfiles ? deepClone(resAllFactorsProfile) : deepClone(resAllFactors);
+      var index = options.singleFactorRes ? 0 : factorEnrollList[options.factorName].index;
+      var factor = res.response._embedded.factors[index];
+      factor.status = options.status;
+      factor.enrollment = options.enrollment;
+      if (options.useProfiles) {
+        var cardinality = {
+          min: options.policyEnrollment.minimum,
+          max: options.policyEnrollment.maximum
+        };
+        factor.profiles[0]._embedded.features[0].cardinality = cardinality;
+        for (var i = 0; i < options.policyEnrollment.enrolled; i++) {
+          factor.profiles[0]._embedded.enrolledFactors.push(options.enrolledFactor);
+        }
+      }
+      else {
+        factor.policy.enrollment = options.policyEnrollment;
```

Comment:
similar to how you changed it in the src code, we can change the name from `policyEnrollment` to `cardinality`